vllm/worker/model_runner.py

```diff
@@ -1386,6 +1392,16 @@ def execute_model(
             logits=logits,
             sampling_metadata=model_input.sampling_metadata,
         )
+        if (self.observability_config.collect_model_forward_time
+                and output is not None):
```

Comment:
```suggestion
        if (self.observability_config
                and self.observability_config.collect_model_forward_time
                and output is not None):
```
To address the CI error:
```
vllm/worker/model_runner.py:1395: error: Item "None" of "Optional[Any]" has no attribute "collect_model_forward_time"  [union-attr]
```
please add a check to ensure `self.observability_config` is not None. Additionally, run `./format.sh` to fix any other linting issues.

Reply:
Done.